app/views/api/v1/api_docs/swagger_ui.html.erb

```diff
@@ -0,0 +1,65 @@
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
```

Comment:
same -- could use a comment at the top explaining source/reason

Reply:
2b00741